src/components/new-safe/CreateSafe/index.tsx

```diff
@@ -22,24 +24,56 @@ export type NewSafeFormData = {
   mobileOwners: NamedAddress[]
 }
 
-export const CreateSafeSteps: TxStepperProps<NewSafeFormData>['steps'] = [
-  {
-    title: 'Select network and name Safe',
-    subtitle: 'Select the network on which to create your Safe',
-    render: (data, onSubmit, onBack) => <CreateSafeStep1 onSubmit={onSubmit} onBack={onBack} data={data} />,
+const staticHints: Record<number, { title: string; variant: AlertColor; steps: { title: string; text: string }[] }> = {
+  0: {
+    title: 'Safe Creation',
+    variant: 'info',
+    steps: [
+      {
+        title: 'Network fee',
+        text: 'Deploying your Safe contract requires the payment of the associated network fee with your connected wallet. An estmation will be provided in the last step.',
```

Comment:
I think we should remove "contract" here to be more user friendly. What do you think?